### PR TITLE
MPC Benchmark Abstraction (sigmoid, tanh)

### PR DESCRIPTION
**Pull request by marload**
## Description

https://github.com/OpenMined/PySyft/issues/4486

I abstracted the Benchmark.
This abstraction has not yet been applied to `ast_benchmark.py`.

## How has this been tested?

`python benchmark_tanh.py`
`python benchmark_sigmoid.py`

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import abc
+
+from .workers_initialization import workers, hook
+
+
+class AbstractBenchmark(metaclass=abc.ABCMeta):
```
Comment (gmuraru) [inline benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py:6]: Q: Can you use ```class AbstractBenchmark(ABC) and do ```from abc import ABC```. There are a few examples in ```syft/generic/abstract```

**Comment by gmuraru** [comment]
It looks great. Great job!

File: benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import abc
+
+from .workers_initialization import workers, hook
```
Comment (gmuraru) [inline benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py:3]: Q: Could you use here the full path (only to be consistent with the rest of the files)

**Review by gmuraru — APPROVED** [review]

File: benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py (hunk 1 above)
Comment (marload) [reply benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py]: Sure!

File: benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py (hunk 2 above)
Comment (marload) [reply benchmarks/frameworks/torch/mpc/scripts/abstract/abstract_benchmark.py]: Sure!

**Comment by aanurraj** [comment]
Great work 😀!

**Review by Syzygianinfern0 — APPROVED** [review]
Thank you for your contribution! :tada:

**Pull request merged**